notebooks/query-language-string-based-dialect/04.02.basic_numeric_comparison.ipynb

```diff
@@ -0,0 +1,575 @@
+       "\n",
+       "Based on the types of indexing (`Index or MultiIndex`), retrieving data from a DataFrame corresponding to a particular node either retrieves a single or multiple rows. This difference can cause issues when applying query node predicates.\n",
+       "Therefore, it is necessary to get rid of all index levels besides the node column through an aggregation operation on the GraphFrame. Then, a query node predicate can be applied to the GraphFrame. "
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# why use drop index levels\n",
+    "display(Markdown(\"../common/documentation/drop-index-information.md\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "f50aec9a-c6a8-420c-8d2d-0ff3d643b92b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gf.drop_index_levels()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "06ce6daf-b98e-4006-b3d4-d56f7cd959c3",
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    },
+    "tags": [
+     "hide-input"
+    ]
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": [
+       "### Query type 1: Evaluate numeric metric comparison (e.g., greater than) in query node predicates\n",
+       "\n",
+       "\n",
+       "In the simplest case, a user can use this query type to find GraphFrame nodes where the time spent on the nodes is comparable to a numerical value using inequalities (<, >, <=, >=). A practical use case of such type of query is to constrain which nodes to match before conducting a numeric metric comparison. Below, we take a look at an example with a similar use case.\n",
+       "\n",
+       "The following query first filters all single nodes with the name metric `lulesh.cycle`. Then, the second query node includes all nodes below the lulesh.cycle node in the GraphFrame. The final query node matches single nodes in the GraphFrame where the numeric comparison is satisfied when the `time` metric `is greater than 100000`:"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# display query type 1 documentation\n",
+    "display(Markdown(\"../common/documentation/basic-numeric-comparison-01.md\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "83c28c99-c418-442f-bbe6-8ec3a577a817",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query_1 = \"\"\"\n",
+    "MATCH (\".\", p)->(\"*\")->(\".\", p1)\n",
```

Comment:
Same issue occurs here with the values being different from the original tree to the filtered tree.